docs/tutorials/classification.ipynb

```diff
@@ -2,195 +2,433 @@
+       "|           |           | Mr.       |           |   |           |           |           |          |\n",
+       "|           |           | Anthony   |           |   |           |           |           |          |\n",
+       "| max       | 1308.0000 | van Melke | male      | … | 512.32920 |         T | Southampt |  1.00000 |\n",
+       "|           |         0 | beke, Mr. |           |   |           |           | on        |          |\n",
+       "|           |           | Philemon  |           |   |           |           |           |          |\n",
+       "| mean      | 654.00000 | -         | -         | … |  33.29548 |         - | -         |  0.38197 |\n",
+       "| median    | 654.00000 | -         | -         | … |  14.45420 |         - | -         |  0.00000 |\n",
+       "| standard  | 378.02006 | -         | -         | … |  51.75867 |         - | -         |  0.48606 |\n",
+       "| deviation |           |           |           |   |           |           |           |          |\n",
+       "| distinct  | 1309.0000 | 1307      | 2         | … | 281.00000 |       186 | 3         |  2.00000 |\n",
+       "| value     |         0 |           |           |   |           |           |           |          |\n",
+       "| count     |           |           |           |   |           |           |           |          |\n",
+       "| idness    |   1.00000 | 0.9984721 | 0.0015278 | … |   0.21543 | 0.1428571 | 0.0030557 |  0.00153 |\n",
+       "|           |           | 16119175  | 838808250 |   |           | 428571428 | 677616501 |          |\n",
+       "|           |           |           | 573       |   |           |         5 | 145       |          |\n",
+       "| missing   |   0.00000 | 0.0       | 0.0       | … |   0.00076 | 0.7746371 | 0.0015278 |  0.00000 |\n",
+       "| value     |           |           |           |   |           |  27578304 | 838808250 |          |\n",
+       "| ratio     |           |           |           |   |           |           | 573       |          |\n",
+       "| stability |   0.00076 | 0.0015278 | 0.6440030 | … |   0.04587 | 0.0203389 | 0.6993114 |  0.61803 |\n",
+       "|           |           | 838808250 | 557677616 |   |           | 830508474 | 001530222 |          |\n",
+       "|           |           | 573       |           |   |           |        56 |           |          |\n",
+       "+-----------+-----------+-----------+-----------+---+-----------+-----------+-----------+----------+"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
    ],
-   "metadata": {
-    "collapsed": false
-   }
+   "source": [
+    "raw_data.summarize_statistics()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We remove certain columns for the following reasons:\n",
+    "1. **high idness**: `name`, `id` , `ticket`\n",
```

Comment:
I'd propose keeping the `name` column but declaring it as an "extra_name" when calling `to_tabular_dataset`. While models must ignore it, it's still helpful when interpreting the predictions of the model.